whois-api/src/main/java/net/ripe/db/whois/api/httpserver/JettyBootstrap.java

```diff
@@ -258,33 +265,51 @@ private Connector createInsecureConnector(final Server server, final HttpConfigu
 
     /**
      * Use the DoSFilter from Jetty for rate limiting: https://www.eclipse.org/jetty/documentation/current/dos-filter.html.
-     * See {@link WhoisDoSFilter} for the customisations added.
-     * @return the rate limiting filter
+     * See {@link DoSLookUpFilter} or {@link DoSUpdateFilter} for the customisations added.
      * @throws JmxException if anything goes wrong JMX wise
      * @throws JMException if anything goes wrong JMX wise
      */
-    private FilterHolder createDosFilter() throws JmxException, JMException {
-        WhoisDoSFilter dosFilter = new WhoisDoSFilter();
-        FilterHolder holder = new FilterHolder(dosFilter);
-        holder.setName("DoSFilter");
-
+    private void createDosFilter(final WebAppContext context) throws JmxException, JMException {
         if (!dosFilterEnabled) {
             LOGGER.info("DoSFilter is *not* enabled");
+            return;
         }
+
+        final String maxRequestPerMsLookUp = "" + 10 * 60 * 1_000; // high default, 10 minutes
+        final FilterHolder lookUpDoSFilter = createDosFilter("DoSLookUpFilter", "50", maxRequestPerMsLookUp, new DoSLookUpFilter());
+
+        final FilterHolder updateDoSFilter = createDosFilter("DoSUpdateFilter", dosUpdatesMax, "" + Integer.parseInt(dosUpdatesMax) * 1_000, new DoSUpdateFilter());
```

Comment:
Magic numbers here, separate them into a variable so it's easier to read.

Reply:
Move arithmetic into `createDosFilter` ?

Reply:
I move all of these to different FilterHolders.
I think that it is much clear if we keep the configuration per each in different classes